pymongo/message.py

```diff
@@ -431,8 +441,12 @@ def get_message(self, dummy0, sock_info, use_cmd=False):
         if use_cmd:
             spec = self.as_command(sock_info)[0]
             if sock_info.op_msg_enabled:
+                if self.exhaust_mgr:
+                    op_msg_flags = _OpMsg.EXHAUST_ALLOWED
```

Comment:
nit: I think `flags` is a better name. This var is immediacy passed to `_op_msg()` so it's clear that they are flags for op_msg.

Reply:
Done.